stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1415,3 +1415,32 @@ function normalize(v::AbstractVector, p::Real = 2)
         return T[]
     end
 end
+
+"""
+    eachrow(A::AbstractVecOrMat)
+
+Create a generator iterating over views of A's first dimension.
+See also [`eachcol`](@ref) and [`eachslice`](@ref).
+"""
+eachrow(A::AbstractVecOrMat) = (view(A, i, :) for i in axes(A, 1))
+
+
+"""
+    eachcol(A::AbstractVecOrMat)
+
+Create a generator iterating over views of A's second dimension.
+See also [`eachrow`](@ref) and [`eachslice`](@ref).
+"""
+eachcol(A::AbstractVecOrMat) = (view(A, :, i) for i in axes(A, 2))
+
+"""
+    eachslice(A; dims = 1)
+
+Create a generator iterating over views of A's nth dimension.
```

Comment:
Whats `n`?

Reply:
Well, you can't really say `over A's dims-th dimension`...

Reply:
How about `over dimension dim of A`?

Reply:
It is good to refer to the argument in the function declaration.